documentation/scripts/checkSize.sh

```diff
@@ -5,7 +5,7 @@ if [ -d "$markdownDir" ]
 then
   echo "$markdownDir is a directory."
 else
-  echo "$markdownDir is not a directory. You need to run from the root of the Detect project (e.g. src/synopsys-detect)"
+  echo "$markdownDir is not a directory. You need to run from the root of the Detect project (e.g. src/blackduck-detect)"
```

Comment:
Should use blackduck-detect for our root project name and jar names for consistency? Right now, it is just detect.

Reply:
The good thing about using just detect.jar is that it we "Black Duck" could be resold at any time?
For the root project name, I can use the full name variable in that sentence, at least it can be swapped out.